### ISSUE TITLE
Save button doesn't update after Back button in Firefox

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-connectors#321]
1) In Firefox, go to Google Scholar. Search for "test".

2) Click the first result.

3) Click Back to the search results.

The button will show the webpage icon rather than the folder icon.

It looks like Firefox doesn't call the 'load' listener in inject.jsx, so [this call to Zotero.Inject.init()](https://github.com/zotero/zotero-connectors/blob/c1f64a0b79d3a5851f6f18546a21cdc654ad92a4/src/common/inject/inject.jsx#L645) doesn't happen. In Chrome it does.

**Comment by adomasven** [comment]
Tested with Fx 68 where this works correctly, broken since at least Fx 78. Seems to be due to back/forward cache. Is fixed by adding

```js
window.addEventListener('unload', function () {});
```

to any content script (as per [this article](https://coderwall.com/p/3t6ila/disable-the-back-forward-cache-of-firefox-and-safari)), although that completely disables the back/forward cache in Firefox completely, which would make our extension quite rogue if we did that.

Created a [bug report on bugzilla](https://bugzilla.mozilla.org/show_bug.cgi?id=1657890) and marking this upstream.
